### ISSUE TITLE
Buildings without heating system but with cooling system can not have conditioned floor area

### ISSUE THREAD

**Issue by gabriel-happle** [architecture-building-systems/CEAforArcGIS#733]
Whether or not buildings have a conditioned floor area `Af` depends only on the heating system.
In Singapore climate buildings are conditioned but do not have heating systems.
The code in `thermal_loads.calc_prop_rc_model()` has to be changed:

```
        for building in df.index.values:
            if hvac_temperatures.loc[building,'type_hs'] == 'T0':
                df.loc[building,'Hs'] = 0
                print 'Building %s has no heating system, Hs corrected to 0.' % building

```

**Comment by martin-mosteiro** [comment]
Hm good catch, that was actually something I added because of an error I found in the HSQ case study. The easy way out I guess would be to have something like `if 'type_hs' == 'T0' and 'type_cs' == 'T0'` then `Hs = 0`. But then again other properties are calculated based on `Af` (e.g. the process heating demand I added the other day :) ) so then you might still have a case where there's no space conditioning but you do have process heating in the building.

Maybe it should be up to the user to define sane values for `Hs` instead of forcing the value.

**Comment by gabriel-happle** [comment]
> But then again other properties are calculated based on Af (e.g. the process heating demand I added the other day :) ) so then you might still have a case where there's no space conditioning but you do have process heating in the building.

**Comment by gabriel-happle** [comment]
solved with #741
